src/templates/Header/index.js

```diff
@@ -1,13 +1,12 @@
-/* eslint react/prop-types: 0 */
-
 import React, { useEffect } from 'react'
 import { connect } from 'react-redux'
+import { bool, func, string } from 'prop-types'
 
 import Preloader from 'ui-kit/preloader/preloader'
 
 import { requestImages } from '../../redux/actions'
-
 import '../scss/Hedaer.scss'
+import Nasa from '../Nasa'
```

Comment:
For me it's very unclear what does Nasa stand for.... is it a component?  a reducer ?  an action ? let's be more clear next time, it's too generic I think.

Reply:
lets choose another name for **Nasa**, its very abstract and unclear, is it a component  ? maybe just **HomePage** instead of **Nasa** ?

Reply:
It is a temporary package for a clearer image